.github/ISSUE_TEMPLATE/--bug-report.md

```diff
@@ -0,0 +1,59 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to improve Alpakka Kafka
+---
+
+### Are you looking for help?
+
+If you have a [Lightbend Subscription](https://www.lightbend.com/lightbend-platform-subscription), please reach out via the [Lightbend Portal](https://portal.lightbend.com/).
+
+This is an issue tracker, used to manage and track the development of **Alpakka Kafka**. 
+
+Please report issues regarding specific projects in their respective issue trackers, e.g.:
+ - Akka:      https://github.com/akka/akka/issues 
+ - Akka HTTP: https://github.com/akka/akka-http/issues 
+ - Alpakka:   https://github.com/akka/alpakka/issues 
+
+This is not a support system, ask questions or discuss ideas in the [Lightbend discuss forum](https://discuss.lightbend.com/c/akka/streams-and-alpakka).
```

Comment:
`please` or `you can` makes it sound less commanding.

Reply:
Yes. The first part could go away entirely.

> Please ask questions or discuss ...